plugins/todo-backend/src/schema/openapi.yaml

```diff
@@ -0,0 +1,127 @@
+openapi: 3.1.0
```

Comment:
Does this need to be `3.0.3`? Editor complained about it so moved to `3.1.0`, didn't seem to break anything so far at least 😁

Reply:
For now, I'd suggest leaving it at 3.0.3. There are some weirdnesses with 3.1 that haven't been ironed out and OSS support, like `openapi-generator`, doesn't yet have it.